components/test_raftstore/src/util.rs

```diff
@@ -149,6 +149,9 @@ pub fn new_server_config(cluster_id: u64) -> ServerConfig {
         // Considering connection selection algo is involved, maybe
         // use 2 or larger value here?
         grpc_raft_conn_num: 1,
+        // Disable stats concurrency. procinfo performs too bad without optimization,
+        // disable it to save CPU for real tests.
+        stats_concurrency: 0,
```

Comment:
could this be a bool rather than an int?

Reply:
It's number controlling how many threads will be spawned to collect runtime stats.

Reply:
> Does it reduce code coverage?

The pool is used for collecting CPU usage and spawn some tasks for raft client. The second part is tested in `integrations/server/raft_client.rs`, so I guess it's OK.